org.opent2t.sample.lamp.superpopular/test/lampTests.js

```diff
@@ -139,10 +139,10 @@ function runLampTests(settings) {
     test.serial('SetColourChroma', t => {
         return helpers.runTest(settings, t, () => {
             return OpenT2T.invokeMethodAsync(translator, SchemaName, 'getDevicesColourChroma', [deviceId]).then((initialTemperature) => {
-                return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesColourChroma', [deviceId, { 'ct': 30 }]).then(() => {
+                return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesColourChroma', [deviceId, { 'ct': 2500 }]).then(() => {
```

Comment:
>return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesColourChroma', [deviceId, { 'ct': 2500 }]).then(() => { [](start = 16, length = 124)

Does this break other lamp tests that use provide colourChroma resources?

Additionally, as I mentioned in my other comment.  Unites here should be Mired, not Kelvin. [http://www.oneiota.org/revisions/1757](http://www.oneiota.org/revisions/1757)